test/test_util.h

```diff
@@ -0,0 +1,16 @@
+// SPDX-License-Identifier: GPL-3.0-or-later
+// Copyright (c) 2021 Scipp contributors (https://github.com/scipp)
+/// @file
+/// @author Neil Vaytet
+#pragma once
+
+#include "scipp/variable/variable.h"
+
+using namespace scipp;
+
+inline Variable arange(const Dim dim, const scipp::index shape) {
+  std::vector<double> values;
+  for (scipp::index i = 0; i < shape; ++i)
+    values.push_back(i);
+  return makeVariable<double>(Dims{dim}, Shape{shape}, Values(values));
+}
```

Comment:
- Bind to Python?
- `shape` -> `size`? (It is 1D)
- Can the vector be moved into the Variable?

Reply:
Variables do not use `std::vector` internally so moving is not possible. I would not bind this to Python with this signature.

Reply:
> I would not bind this to Python with this signature.

Yes. But maybe we should add it separately.